Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  KlaviyoAPI.swift
+//  
+//
+//  Created by Noah Durell on 11/8/22.
+//
+
+import Foundation
+
+
+struct KlaviyoAPI {
+    struct KlaviyoRequest {
+      enum KlaviyoEndpoint {
+          struct CreateProfilePayload: Encodable {
+              /**
+               Internal structure which details not needed by the public API.
+               */
+              struct Profile: Encodable {
```

Comment:
can this be achieved without this nesting structure? does `internal` exist as a keyword? i feel like this file really long... if we add a new endpoint that would have its own struct Payload and continues to add to this file

Reply:
I'm happy to move it to it's own file! :)

Reply:
Everything that isn't public should be internal.

Reply:
Moved to a new file internal models. Still nested under the old one.